varcode/effect_collection.py

```diff
@@ -262,3 +263,26 @@ def gene_counts(self):
         for effect in self:
             counter[effect.gene_name] += 1
         return counter
+
+    def to_dataframe(self):
+        """Build a dataframe from the effect collection"""
+        def row_from_effect(effect):
+            row = {}
+            row['chr'] = effect.variant.contig
```

Comment:
I think `contig` might be better than `chr`. Also, I'd suggest including the reference genome in each row (e.g. a column called `genome` that gets set to `grch37` in every row)